### PR TITLE
[IMP] *_*: update tooltip in various modules

### PR DESCRIPTION
**Pull request by dhba-odoo**
In this commit we rewrite some tooltips because some of them are not
correct in English and others are not valid anymore.Add some tooltips that
could be missing and remove some tooltips that are not valid anymore.

### CODE REVIEW

File: addons/rating/models/rating.py [hunk 1]
```diff
@@ -37,20 +37,19 @@ def _selection_target_model(self):
...
-    feedback = fields.Text('Comment', help="Reason of the rating")
+    feedback = fields.Text('Comment')
     message_id = fields.Many2one(
         'mail.message', string="Message",
-        index=True, ondelete='cascade',
-        help="Associated message when posting a review. Mainly used in website addons.")
-    is_internal = fields.Boolean('Visible Internally Only', readonly=False, related='message_id.is_internal', store=True)
-    access_token = fields.Char('Security Token', default=_default_access_token, help="Access token to set the rating of the value")
+        index=True, ondelete='cascade')
+    is_internal = fields.Boolean('Visible Internally Only', readonly=False, related='message_id.is_internal', store=True, help="")
```
Comment (IT-Ideas) [inline addons/rating/models/rating.py:51]:
```suggestion
    is_internal = fields.Boolean('Visible Internally Only', readonly=False, related='message_id.is_internal', store=True)
```

**Review by IT-Ideas — COMMENTED** [review]
Thanks 🙂 
Thanks @LNAodoo for having checked!

Just one remark.

File: addons/rating/models/rating.py (hunk 1 above)
Comment (mba-odoo) [reply addons/rating/models/rating.py]: @IT-Ideas , Done!!

**Review by IT-Ideas — APPROVED** [review]
Thanks 🙂 
@robodoo r+

**Pull request closed without merging**